snippets

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:20.04
+
+# Python version.
+ARG py_ver=3
+
+RUN apt update \
+    # Dependencies.
+    && apt -y install python${py_ver} python${py_ver}-distutils curl \
+    # Install pip.
+    && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py \
```

Comment:
Is this the preferred/recommended way to install `pip` vs `apt install`?

Reply:
Definitely `apt install` is first-recommended way.
However, from my previous experiences, doing so wasn't always successful at installing pip, especially for python version 3. So I instead started following [this guide].(https://pip.pypa.io/en/stable/installing/).
I haven't tried if `apt install` works in Ubuntu 20.04 without any error though..

Reply:
Just run a quick test:
- `sudo apt install python3-pip` works.
- `sudo apt install python-pip` fails with error: `E: Unable to locate package python-pip` - [see this](https://superuser.com/questions/1545380/ubuntu-20-04-e-unable-to-locate-package-python-pip)

Reply:
Good news is that even with Python2 being deprecated from Ubuntu 20.04, the above alternative way works okay, with little extra revision.